src/Sql/Sql/help/Get-AzSqlInstancePool.md

```diff
@@ -0,0 +1,214 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version:
```

Comment:
@johnpaulkee please make sure that all of the new cmdlet markdown files you're adding have the correct URL for the `online version` field in the header of the file -- the build is currently failing since this value is missing from the new files. You can find what the exact URL should be from the artifacts of the CI.